raspberry/bzzz/sensors/gps.py

```diff
@@ -0,0 +1,279 @@
+import serial
+import time
+from threading import Thread, Lock
+import csv
+from datetime import datetime, timedelta
+
+def deg_min_sec_to_decimal(degrees, minutes, direction):
+    """
+    Convert degrees-minutes-seconds from GNGLL to decimal
+
+    :param degrees: degrees
+    :param minutes: minutes
+    :param direction: direction (can be one of 'N', 'W', 'E', 'S')
+    :return: decimal representation of GPS coordinates
+    """
+    decimal_degrees = degrees + minutes / 60
+    if direction in ['S', 'W']:  # need to be denoted with a negative value
+        decimal_degrees *= -1
+    return decimal_degrees
+
+class GpsLogger:
```

Comment:
Why don't you use the [other logger](https://github.com/QUB-ASL/bzzz/blob/dev/raspberry/bzzz/sensors/data_logger.py) we already have?